src/components/WearablePreview/WearablePreview.tsx

```diff
@@ -0,0 +1,28 @@
+import * as React from 'react'
+import './WearablePreview.css'
+
+export type WearablePreviewProps = {
+  contractAddress: string
+  tokenId?: string
+  itemId?: string
+}
```

Comment:
What if we change the type to something like:
```
export type WearablePreviewProps =
  | WearablePreviewPropsWithTokenId
  | WearablePreviewWithItemId

type BaseProps = {
  contractAddress: string
}

type WearablePreviewPropsWithTokenId = BaseProps & {
  tokenId: string
  itemId?: never
}

type WearablePreviewWithItemId = BaseProps & {
  itemId: string
  tokenId?: never
}
```
to prevent compiling the WearablePreview component with both tokenId and itemId?

Reply:
done! ✅